### DISCUSSION TITLE
skeleton height

### DISCUSSION THREAD

**Discussion by AlexDevMakarov** [dvtng/react-loading-skeleton#150] [General]
Hello,i need your help if you don't mind, please
i can't figure out with height
how did you make height took font's size if we did't attached height props?
thank you in advance

**Comment by srmagura** [comment]
The `<Skeleton>` should automatically use the same font size that would be applied to normal text. If this isn't working, can you create a CodeSandbox that shows the problem? It is difficult to debug the issue without more context. Thanks.
